presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewQueryOptimizer.java

```diff
@@ -366,13 +364,15 @@ private QuerySpecification getRewrittenQuerySpecification(Metadata metadata, Qua
     private class QuerySpecificationRewriter
             extends AstVisitor<Node, Void>
     {
+        Logger log = Logger.get(QuerySpecificationRewriter.class);
```

Comment:
static final, and newline after this

Reply:
Can't be static because it's in an inner class. I just removed this and am using the logger from `MaterializedViewQueryOptimizer` instead.